test/e2e/installplan_e2e_test.go

```diff
@@ -340,7 +340,6 @@ var _ = Describe("Install Plan", func() {
 			mainPackageStable := fmt.Sprintf("%s-stable", mainPackageName)
 			mainPackageBeta := fmt.Sprintf("%s-beta", mainPackageName)
 			dependentPackageStable := fmt.Sprintf("%s-stable", dependentPackageName)
-			dependentPackageBeta := fmt.Sprintf("%s-beta", dependentPackageName)
```

Comment:
This test is designed to test the CRD ownership for an operator that is replaced by a newer version. The dependent operator is not needed because we only test the main operator, not the dependent one. I wrote the test case and not sure why I added the dependent operator in the first place.
The older resolution would pick the older version of dependent operator but with the new resolution, it will pick the latest in the channel that provides the requiredAPIs. As a result, the current test will fail. I simply remove one version of dependent operator to address the issue.